vnext/ReactWindowsCore/Modules/UIManagerModule.cpp

```diff
@@ -271,11 +273,18 @@ void UIManager::setChildren(int64_t viewTag, folly::dynamic &&childrenTags) {
     auto &childNode = m_nodeRegistry.getNode(tag);
     childNode.m_parent = parent.m_tag;
     parent.m_children.push_back(tag);
-    if (!parent.m_zombie)
-      parent.AddView(childNode, index);
-
-    m_nativeUIManager->AddView(parent, childNode, index);
-
+    try {
+      if (!parent.m_zombie)
+        parent.AddView(childNode, index);
+
+      m_nativeUIManager->AddView(parent, childNode, index);
+    } catch (const winrt::hresult_error &e) {
+      if (e.code() == E_NOINTERFACE) {
+        DEBUG_HRESULT_ERROR(e);
+      } else {
+        throw;
```

Comment:
Should this be `winrt::throw_hresult(e)`? Or does `throw` automatically does that?

Reply:
`winrt::throw_hresult` takes an `HRESULT` (i.e. an integer) and constructs an instance of `winrt::hresult_error` and throws it (with the stack starting at that point). 
The `throw` keyword rethrows the exception that was caught in the current `catch` block with the existing stack.